### ISSUE TITLE
Compile Error - Freebayes Upgrade

### ISSUE THREAD

**Issue by kelvin-wcl** [tseemann/snippy#136]
Hi Thorsten,

I have installed Gubbins and Snippy. When I tried to execute snippy, I received an error regarding upgrading Freebayes:

Can't exec "freebayes": No such file or directory at /home/ubuntu/snippy/bin/snippy line 80.
readline() on closed filehandle $VER at /home/ubuntu/snippy/bin/snippy line 81.
Use of uninitialized value $ver in numeric lt (<) at /home/ubuntu/snippy/bin/snippy line 85.
[22:35:37] Need freebayes --version >= 1.1 - please upgrade it.

I have tried to install Freebayes but I still receive the same error (git clone --recursive git://github.com/ekg/freebayes.git)

Please advise. Thank you.

Kind Regards,
Kelvin

**Comment by tseemann** [comment]
What happens if you type `freebayes --version` ?

This is mine:
```
freebayes --version
version:  v1.1.0-dirty
```

**Comment by kelvin-wcl** [comment]
Hi Torsten,

I get this:

freebayes --version
freebayes: command not found


Snippy has been installed:
snippy --help
Synopsis:
  snippy 3.2-dev - fast bacterial variant calling from NGS reads
Author:
  Torsten Seemann <torsten.seemann@gmail.com>

Thank you.

Kind Regards,
Kelvin

**Comment by kelvin-wcl** [comment]
Hi Torsten,
What other software can I use to generate the alignment file for input to Gubbins besides Snippy? Thanks.



University of Tasmania Electronic Communications Policy (December, 2014).
This email is confidential, and is for the intended recipient only. Access, disclosure, copying, distribution, or reliance on any of it by anyone outside the intended recipient organisation is prohibited and may be a criminal offence. Please delete if obtained in error and email confirmation to the sender. The views expressed in this email are not necessarily the views of the University of Tasmania, unless clearly intended otherwise.

**Comment by tseemann** [comment]
You can solve the github error using:

```
git clone --recursive https://github.com/ekg/freebayes.git)
```

ie. replace the git:// with https://

I'm working on updating the included freebayes binary to 1.1 version.

**Comment by tseemann** [comment]
If you use the latest `git clone` of Snippy, i've updated the `freebayes` binary.
